docs/users/developer-tools.md

```diff
@@ -28,6 +28,16 @@ To execute a fully-featured `composer create-project` command, you can execute t
 
 `ddev exec composer create-project ...`
 
+#### Windows OS and `ddev composer`
```

Comment:
Came for instructions on how to enable developer mode, but there aren't any!

Reply:
Search for "developer" in settings. Thanks for pointing this out :) But if you want to continue it's not hard.

Reply:
I did eventually find it, but wasn't sure if it was a Docker setting or a Windows setting. Just referring to this:

`please see the docs to enable developer mode: https://ddev.readthedocs.io/en/latest/users/developer-tools/#windows-os-and-ddev-composer`

Reply:
See if this works for you, https://github.com/drud/ddev/blob/ab4965c15cdc5b893e2224deef4ebe136dcef729/docs/users/developer-tools.md#windows-os-and-ddev-composer

Reply:
Woah, screenshots and everything! Awesome.